director/director.go

```diff
@@ -1062,17 +1063,24 @@ func registerServeAd(engineCtx context.Context, ctx *gin.Context, sType server_s
 		}
 	}
 
+	st := adV2.StorageType
+	if st == "" {
+		st = server_structs.OriginStoragePosix
+	}
+
 	sAd := server_structs.ServerAd{
-		Name:        adV2.Name,
-		URL:         *adUrl,
-		WebURL:      *adWebUrl,
-		BrokerURL:   *brokerUrl,
-		Type:        sType,
-		Caps:        adV2.Caps,
-		Writes:      adV2.Caps.Writes,
-		DirectReads: adV2.Caps.DirectReads,
-		Listings:    adV2.Caps.Listings,
-		IOLoad:      0.5, // Defaults to 0.5, as 0 means the server is "very free" which is not necessarily true
+		Name:                adV2.Name,
+		StorageType:         st,
```

Comment:
What happens when it's a V1 version of the ads and the storage type is checked and or DisableDirectorTest? Will the break backwards compatibility? Can we test for this?